src/gpuarray_buffer_opencl.c

```diff
@@ -90,9 +90,10 @@ cl_ctx *cl_make_ctx(cl_context ctx) {
   res->refcnt = 1;
   res->exts = NULL;
   res->blas_handle = NULL;
-  res->q = clCreateCommandQueue(ctx, id,
-				qprop&CL_QUEUE_OUT_OF_ORDER_EXEC_MODE_ENABLE,
-				&err);
+  res->q = clCreateCommandQueue(
```

Comment:
I don't need to keep the flags around for anything in the OpenCL backend.  If there is ever a need for that, then I'll add the field and fill it.